packages/webdriver/protocol/appium.json

```diff
@@ -944,10 +944,13 @@
       "description": "Set GSM signal strength (Emulator only).",
       "ref": "http://appium.io/docs/en/commands/device/network/gsm-signal/",
       "parameters": [{
+        "name": "signalStrength",
+        "type": "string",
+        "description": "signal strength in the range [0, 4]"
```

Comment:
the `signalStrength` parameter should be required and in the description it should say that this parameter works with Appium vX upwards and the other has to be set if user is using a lower version

Reply:
Thanks @christian-bromann I addressed your comment.

Reply:
@christian-bromann The PR https://github.com/appium/appium-base-driver/pull/308 has been merged.